dashboard/app/controllers/courses_controller.rb

```diff
@@ -189,6 +143,53 @@ def i18n_params
 
   private
 
+  def get_unit_group
+    course_name = params[:course_name]
+
+    unit_group =
+      params[:action] == "edit" ?
+        UnitGroup.get_without_cache(course_name) :
+        UnitGroup.get_from_cache(course_name)
```

Comment:
I think you may be able to remove this -- the edit route will only be accessible in levelbuilder_mode, in which case get_from_cache will return the uncached version because of these lines: https://github.com/code-dot-org/code-dot-org/blob/c663c3f1f64b6d7bdc428f59a9ad73bbcc733be8/dashboard/app/models/script.rb#L436-L437

Reply:
Great - i will update